docker-compose.template.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   graph-node:
-    image: graphprotocol/graph-node:v0.25.0
+    image: graphprotocol/graph-node:v0.25.2
```

Comment:
Why is the graph-node updated?

Reply:
why was it not .2 in the first place?
this a minor version, no breaking changes are expected
I think it is better to use it